modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
That's a great idea.  I didn't know about that.  If I do it this way though:

````scala
    (for {
      _ <- ProgramDao.insert(p).onUniqueViolation(0.point[ConnectionIO])
      _ <- ObservationDao.insert(o).onUniqueViolation(0.point[ConnectionIO])
    } yield ()).transact(lxa)
````

I get an exception: 

    org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block

If I `transact` each individual insert though, it works:

````scala
    for {
      _ <- ProgramDao.insert(p).onUniqueViolation(0.point[ConnectionIO]).transact(lxa)
      _ <- ObservationDao.insert(o).onUniqueViolation(0.point[ConnectionIO]).transact(lxa)
    } yield ()
````

This was surprising to me for some reason.  Am I doing it wrong?